package.json

```diff
@@ -1,12 +1,16 @@
 {
   "name": "iobroker.pvforecast",
-  "version": "2.3.0",
+  "version": "2.3.1",
```

Comment:
Please do not update the version manually - the release script will do that.
This is the best example why you shouldn't change it manually: At the moment this version doesn't match io-package.json